otwcode/otwarchive
# AO3-5688 Use double quotes and translatable text on delete user preview page

## Issue

https://otwarchive.atlassian.net/browse/AO3-5688

## Purpose

* Use a consistent translation method on the users/delete_preview view 
* Use double quotes in the option `Change my pseud to "orphan" and attach to the orphan account`
* Update the `When I choose...` step definition to work when the radio button label text has double quotes in it

## Testing Instructions

* Make sure there are double quotes in that option
* Make sure it doesn't say "translation missing" anywhere on the page
